test/tables.iseries.sql

```diff
@@ -1,25 +1,25 @@
-SET CURRENT SCHEMA = CURRENT USER;
+SET CURRENT_SCHEMA = CURRENT USER;
```

Comment:
I have no idea what is this SQL file for. It would be great to provide at least some minimal documentation for people maintaining and contributing to this module.